src/Components/src/Microsoft.AspNetCore.Components.Razor.Extensions/EventHandlerTagHelperDescriptorProvider.cs

```diff
@@ -124,9 +115,10 @@ private List<TagHelperDescriptor> CreateEventHandlerTagHelpers(List<EventHandler
                 {
                     rule.TagName = "*";
 
+                    var entry1 = entry;
```

Comment:
Is this declaration needed?

Reply:
No it is not.

These files were deleted earlier today. Waiting to know if they were migrated to a new project.